tmk_core/protocol/usb_descriptor.h

```diff
@@ -307,5 +329,6 @@ enum usb_endpoints {
 #define CDC_EPSIZE 16
 #define JOYSTICK_EPSIZE 8
 #define DIGITIZER_EPSIZE 8
+#define LAMPARRAY_EPSIZE 32
```

Comment:
```suggestion
#define LAMPARRAY_EPSIZE 8
```
If we don't have any input or output reports, we may decrease the endpoint buffer size to the minimum.